vowpalwabbit/fb_parser/tests/flatbuffer_parser_test.cc

```diff
@@ -28,40 +28,85 @@ flatbuffers::Offset<VW::parsers::flatbuffer::ExampleRoot> sample_flatbuffer_coll
 {
   std::vector<flatbuffers::Offset<VW::parsers::flatbuffer::Example>> examples;
   std::vector<flatbuffers::Offset<VW::parsers::flatbuffer::Namespace>> namespaces;
-  std::vector<flatbuffers::Offset<VW::parsers::flatbuffer::Feature>> fts;
+  // std::vector<flatbuffers::Offset<VW::parsers::flatbuffer::Feature>> fts;
 
   auto label = get_label(builder, label_type);
 
-  fts.push_back(VW::parsers::flatbuffer::CreateFeatureDirect(builder, "hello", 2.23f, constant));
-  namespaces.push_back(VW::parsers::flatbuffer::CreateNamespaceDirect(builder, nullptr, constant_namespace, &fts, 128));
+  // fts.push_back(VW::parsers::flatbuffer::CreateFeatureDirect(builder, "hello", 2.23f, constant));
+  std::vector<flatbuffers::Offset<flatbuffers::String>> feature_names = {builder.CreateString("hello")};
+  std::vector<float> feature_values = {2.23f};
+  std::vector<uint64_t> feature_hashes;// = {constant};
```

Comment:
In parse_example_flatbuffer, you test for nullptr for fields.  Will that work if you pass in empty vectors?   
i.e. does if(ns->feature_names() != nullptr) work if feature_names is an empty vector?

Reply:
I was assuming we would be using vw_to_flat.cc to to convert data to flatbuff. Will add the check.